recipes/espeak-ng/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "espeak-ng" %}
+{% set version = "1.51.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/espeak-ng/{{ name }}/archive/cb62d93fd7b61d8593b9ae432e6e2a78e3711a77.zip
+  sha256: 81f01cdbc928cefde3989e62ee185285c46659503ca0dc1c81e5de2962afd6ad
+
+build:
+  number: 0
```

```suggestion
build:
  number: 0
  skip: true  # [win]
```
Looks like in the latest dev version, the project has switched from autotools to CMake. Maybe skip Windows until the next release?

---

I tried getting this running on Windows over the weekend, while I figured out it for `pcaudiolib` (it was a total disaster) I still couldn't get it for `espeak-ng`.